### PR TITLE
Prepare release v0.1.29

### PR DESCRIPTION
**Pull request by timoreimann**

### CODE REVIEW

**Review by adamwg — APPROVED** [review]
:ship:

**Review by waynr — APPROVED** [review]

File: releases/v0.1.29.yml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+kind: ClusterRole
+metadata:
+  annotations:
+    rbac.authorization.kubernetes.io/autoupdate: "true"
+  name: system:cloud-controller-manager
+rules:
+# The following is necessary to support leader election when running DO-CCM on
+# a multi-master cluster. We leave it commented out on the assumption that the
+# typical user is not running a multi-master cluster and therefore does not
+# need the additional persmissions.
```
Comment (MorrisLaw) [inline releases/v0.1.29.yml:68]: nit: typo `persmissions` should be `permissions`

**Review by MorrisLaw — APPROVED** [review]
lgtm, left a typo nit. But not a blocker!

File: releases/v0.1.29.yml (hunk 1 above)
Comment (timoreimann) [reply releases/v0.1.29.yml]: Good catch, fixed. 👍

**Pull request merged**